### PR TITLE
doc(redis instrumentation): Use latest options in readme

### PR DESCRIPTION
**Pull request by miry**
There is no option `enable_statement_obfuscation` for
`OpenTelemetry::Instrumentation::Redis`.
Replace example with option `db_statement` in README.

### CODE REVIEW

**Review by SomalianIvan — APPROVED** [review]

File: instrumentation/redis/README.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ end
 OpenTelemetry::SDK.configure do |c|
   c.use 'OpenTelemetry::Instrumentation::Redis', {
     # The obfuscation of arguments in the db.statement attribute is enabled by default.
-    # To disable, set enable_statement_obfuscation to false.
-    enable_statement_obfuscation: true,
+    # To disable, set db_statement to :include or :omit if the tag not required.
```
Comment (fbogsany) [inline instrumentation/redis/README.md:50]: Obfuscate is also a valid and supported option.
```suggestion
    # To disable, set db_statement to :include, :obfuscate or :omit if the tag not required.
```

Comment (miry) [reply instrumentation/redis/README.md]: A bit rephrased the comment - I split options to separate lines.

**Review by ericmustin — APPROVED** [review]

File: instrumentation/redis/README.md [hunk 2]
```diff
@@ -47,8 +47,10 @@ end
 OpenTelemetry::SDK.configure do |c|
   c.use 'OpenTelemetry::Instrumentation::Redis', {
     # The obfuscation of arguments in the db.statement attribute is enabled by default.
-    # To disable, set enable_statement_obfuscation to false.
-    enable_statement_obfuscation: true,
+    # To show full query, set db_statement to :include.
+    # To obfuscation, set db_statement to :obfuscate.
+    # To remove tag, set db_statement to :omit
```
Comment (fbogsany) [inline instrumentation/redis/README.md:52]:
```suggestion
    # To include the full query, set db_statement to :include.
    # To obfuscate, set db_statement to :obfuscate.
    # To omit the attribute, set db_statement to :omit.
```

**Review by fbogsany — APPROVED** [review]
Thanks @miry

File: instrumentation/redis/README.md (hunk 2 above)
Comment (miry) [reply instrumentation/redis/README.md]: nice. thank you for the correction. now it sounds better.

**Pull request merged**